code/game/machinery/computer/communications.dm

```diff
@@ -441,12 +441,16 @@
 				to_chat(usr, span_warning("There is no safe code to deliver to your station!"))
 				return
 
+			// ORBSTATION: Notify admins if the person requesting the safe code has an antag datum or a spsecial role
+			if(usr.mind && (usr.mind.special_role || usr.mind.antag_datums?.len > 0))
+				message_admins("[usr] requested the spare ID safe code, but may be an antagonist!")
```

Comment:
This seems like it could be made modular... but I guess if you're already touching this file anyway there's probably not much need.